src/cocaine-app/planner/__init__.py

```diff
@@ -510,16 +591,29 @@ def weight(keys_diff, ts_diff):
 
         return [candidate[0] for candidate in candidates[-count:]]
 
-    def update_recover_ts(self, couple_id, ts):
-        ts = int(ts)
+    def update_historic_ts(self, couple_id, recover_ts=None, cleanup_ts=None):
+        """
+        Update records in mongo with corresponding times
+        :param couple_id: couple_id
+        :param recover_ts: epoch time if we need to update recover_ts time else None
+        :param cleanup_ts: epoch time if we need to update ttl_cleanup time else None
+        """
+
+        updated_values = {'recover_ts': int(recover_ts)} if recover_ts else {'cleanup_ts': int(cleanup_ts)}
```

Comment:
Fixed in https://github.com/yandex/mastermind/pull/366/commits/4cc3dfb4378c8308c3b781c6d8a4c0b731d2a38d